jkube-kit/build/service/jib/src/test/java/org/eclipse/jkube/kit/service/jib/JibServiceUtilTest.java

```diff
@@ -108,48 +103,48 @@ public void testAppendOriginalImageNameTagIfApplicable() {
     }
 
     @Test
-    public void testGetFullImageNameWithDefaultTag() {
+    void testGetFullImageNameWithDefaultTag() {
         assertThat(JibServiceUtil.getFullImageName(getSampleImageConfiguration(), null))
             .isEqualTo("test/test-project:latest");
     }
 
     @Test
-    public void testGetFullImageNameWithProvidedTag() {
+    void testGetFullImageNameWithProvidedTag() {
         assertThat(JibServiceUtil.getFullImageName(getSampleImageConfiguration(), "0.0.1"))
             .isEqualTo("test/test-project:0.0.1");
     }
 
     @Test
-    public void layers_withEmptyLayers_shouldReturnEmpty() {
+    void layers_withEmptyLayers_shouldReturnEmpty() {
         // When
         final List<FileEntriesLayer> result = JibServiceUtil.layers(null, Collections.emptyMap());
         // Then
         assertThat(result).isNotNull().isEmpty();
     }
 
     @Test
-    public void layers_withMultipleLayers_shouldReturnTransformedLayers() throws IOException {
+    void layers_withMultipleLayers_shouldReturnTransformedLayers(@TempDir File temporaryFolder) {
         // Given
         final BuildDirs buildDirs = new BuildDirs("layers-test", JKubeConfiguration.builder()
             .outputDirectory("target/docker")
-            .project(JavaProject.builder().baseDirectory(temporaryFolder.getRoot()).build())
+            .project(JavaProject.builder().baseDirectory(temporaryFolder).build())
             .build());
         final Map<Assembly, List<AssemblyFileEntry>> originalLayers = new LinkedHashMap<>();
         originalLayers.put(Assembly.builder().id("layer-1").build(), Arrays.asList(
-            AssemblyFileEntry.builder().source(temporaryFolder.newFile())
+            AssemblyFileEntry.builder().source(temporaryFolder)
```

Comment:
This change makes the test misleading.

`temporaryFolder.newFile()` statements should be replaced with actual temporary files (you've already done this in other tests)

Reply:
Do I need to do this
```java
File.createTempFile("junit5", null, temporaryFolder)
```
or this?
```java
new File(temporaryFolder, "")
```

Reply:
You should use 

`File.createTempFile("junit5", null, temporaryFolder)` which creates an actual empty file.

Also note that `new File(temporaryFolder, "")` should have a file name, but in this case is the least appropriate option because that doesn't create an actual file (it's just a reference to the file)